pkg/kubelet/types/constants.go

```diff
@@ -21,3 +21,21 @@ const (
 	// system default DNS resolver configuration
 	ResolvConfDefault = "/etc/resolv.conf"
 )
+
+// HairpinMode denotes how the kubelet should configure networking to handle
+// hairpin packets.
+type HairpinMode string
+
+// Enum settings for different ways to handle hairpin packets.
+const (
+	// Set the hairpin flag on the veth of containers in the respective
+	// container runtime.
+	HairpinVeth = "hairpin-veth"
+	// Make the container bridge promiscuous. This will force it to accept
+	// hairpin packets, even if the flag isn't set on ports of the bridge.
+	HairpinBridge = "promiscuous-bridge"
```

Comment:
hairpin-veth and promiscusou-bridge both fit the format of "actual name of flag" - "name of network device that understands that flag", afaik you can only set hairpin on a port ona bridge so hairpin-bridge would break that pattern. Or do you mean the name of the variable? I think having hairpin in the var name denotes its use case, because promiscusousBridge could be a totally unrelated feature. thoughts?